ci/test.sh

```diff
@@ -118,3 +118,77 @@ assert_no_diff
 
 # Run the integration tests
 (cd bindgen-integration && cargo test $CARGO_ARGS)
+
+if [ "$BINDGEN_RUST_FOR_LINUX_TEST" == "1" ]; then
+  # Run the Rust for Linux test
+  #
+  # This is intended to be an end-to-end test that runs what Linux kernel
+  # developers/users would do. It is a quick, build-only test of the Rust code
+  # in the Linux kernel.
+
+  # Put LLVM binaries in the path for `LLVM=1`. The LLVM `bin` directory should
+  # go first since there are others in the Ubuntu image.
+  export PATH="${LLVM_DIRECTORY}/bin:${PATH}"
+
+  # Kernel build dependency: `bindgen-cli`, which is under test.
+  #
+  # Using the unstable `--out-dir` so that we can use `cargo build` instead of `cargo install`,
+  # since we want to reuse the `$CARGO_ARGS`.
+  (cd bindgen-cli && RUSTC_BOOTSTRAP=1 cargo build -Zunstable-options --out-dir ${HOME}/.bindgen $CARGO_ARGS)
+  export PATH="${HOME}/.bindgen:${PATH}"
```

Comment:
If someone knows a better way to handle this that includes `--target` and debug/release, please let me know!

Reply:
Also, I checked in one of my runs that we are actually using the just-built `bindgen`, but it may be nice to double-check somehow that we are actually using that one and not the other that is already installed in `.cargo`. I didn't see e.g. the Git hash printed by `bindgen` in `--version --verbose`, but that could be a nice way (like in `rustc`).